integration-tests/features/steps/common.py

```diff
@@ -71,9 +73,10 @@ def migrate_vm_as_macrocontainer(context, source_vm, target_vm, migration_opt=No
     context.migration_target = target_vm
     context.migration_migration_opt = migration_opt
     migrate_app = context.cli_helper.migrate_as_macrocontainer
-    output = migrate_app(source_vm, target_vm, migration_opt)
+    output = migrate_app(source_vm, target_vm, migration_opt)  # nopep8
```

Comment:
What's `pycodestyle` complaining about here? If it's something we think is noise, I prefer to deal with that by providing suitable config files that turn off particular warnings globally, rather than by selectively turning off all warnings for particular lines.